src/common_modules/vsts-cli-package-common/vsts/cli/package/common/artifacttool.py

```diff
@@ -0,0 +1,98 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import io
+import json
+import logging
+import os
+
+from knack.util import CLIError
+
+from vsts.cli.common.services import _get_credentials
```

Comment:
@tedchamb is _get_credentials meant to be private? I see that it's used also in vsts_git_url_info.py; should we remove the underscore?

Reply:
if you want to remove the underscore that is fine.  I added it to discourage its use, but there are legitimate cases it is needed.